src/Language/AST/DirectiveDefinitionNode.php

```diff
@@ -17,4 +17,10 @@ class DirectiveDefinitionNode extends Node implements TypeSystemDefinitionNode
 
     /** @var NodeList<NameNode> */
     public NodeList $locations;
+
+    public function __construct(array $vars)
+    {
+        parent::__construct($vars);
+        $this->arguments = $this->arguments ?? new NodeList([]);
```

Comment:
```suggestion
        $this->arguments ??= new NodeList([]);
```

Reply:
Ah, forgot that was a thing, thank you ece7502bd75ac0274fa51a6f65ffe72691881a04